netweaver/defaults.yaml

```diff
@@ -11,3 +11,8 @@ netweaver:
       directory: /var/lib/swap
       # Swap size in MB
       size: 2048
+  # by default the exporter is disabled
+  sap_host_exporter:
+      enabled: false
+      sap_control_user: ""
```

Comment:
I wouldn't put the user and password with empty default options. I would remove them. They will never be empty strings, so we have to require them. 
In fact, if `enabled` is false they are not used.

Reply:
In fact, I wouldn't require these 2 options. These parameters are already available in the pillar file in `sid_adm_password` and that `sap_control_user` is created by `{sid}adm` and the sid is available in all nodes pillar file. Use them, the less new parameters the better.

Reply:
ok will check thx